packages/app/src/Element/Bookmarks.tsx

```diff
@@ -19,7 +18,7 @@ const Bookmarks = ({ pubkey, bookmarks, related }: BookmarksProps) => {
   const [onlyPubkey, setOnlyPubkey] = useState<HexKey | "all">("all");
   const loginPubKey = useSelector((s: RootState) => s.login.publicKey);
   const ps = useMemo(() => {
-    return [...new Set(bookmarks.map(ev => ev.pubkey))];
+    return [...new Set(bookmarks.filter(ev => ev.kind === EventKind.TextNote).map(ev => ev.pubkey))];
```

Comment:
Maybe we should look upstream as to why we have these events in here? This could be a bug from the refactor, check in the `useBookmarksFeed` or one of those feeds for bookmarks to see why we're getting likes

Reply:
On it now

Reply:
we seem to be filtering the Pinned feed also in the same way as I did

https://github.com/v0l/snort/blob/ecee50caa94e55b1e4dc9b666aa719290547a1ff/packages/app/src/Pages/ProfilePage.tsx#L229

From my research, it looks like we need the non TextNote events to display the number of likes, zap, etc on the card

Reply:
Yea seems like we should split the lists here: https://github.com/v0l/snort/blob/ecee50caa94e55b1e4dc9b666aa719290547a1ff/packages/app/src/Pages/ProfilePage.tsx#L94

`related` is always for reactions etc so the filtering can be added here, or change the `Bookmarks` element so that it only accepts `bookmarks` and not `related` and continue to do the filtering inside

Reply:
Done. Moved the filtering to `ProfilePage.tsx`